### DISCUSSION TITLE
2.2 Version Available in the NuGet Package?

### DISCUSSION THREAD

**Discussion by rwalshAMNH** [ElmahCore/ElmahCore#177] [Q&A]
Currently only seeing version 2.1 as the latest release in NuGet.

**Comment by kenbailey** [comment]
@rwalshAMNH, I was also wondering about this, see discussion #187, the https://github.com/jrsearles/Elmah.AspNetCore fork looks promising. I've been watching and it's been active, I haven't switched our code bases to it yet though.
